ce_discovery.yaml

```diff
@@ -114,32 +80,91 @@ paths:
     delete:
       operationId: deleteService
       parameters:
-       - in: path
-         name: id
-         description: The id of the service to be returned
-         required: true
-         schema:
-           $ref: '#/components/schemas/service/properties/id'
+        - $ref: '#/components/parameters/id'
       requestBody:
-        required: false
-        content:
-          application/json:
-            schema:
-              $ref: '#/components/schemas/serviceinstance'
+        $ref: '#/components/requestBodies/servicedeleterequest'
       responses:
         '200':
-          description: The Service that was deleted
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/service'
+          $ref: '#/components/responses/servicedeleteresponse'
         '400':
           description: Bad Request - constraint failure
         '404':
           description: Not Found
         '409':
           description: Conflict - epoch not greater
+  /changes:
```

Comment:
We may want to remove this and everything in the commit that adds it.  See discussion in the spec.